src/templates/modules/FM_Template_v1.sol

```diff
@@ -0,0 +1,196 @@
+ *          - Pre-defined layout for all contract functions, modifiers, state
+ *            variables etc.
+ *          - Use of the ERC165Upgradeable contract to check for interface
+ *            support.
+ *
+ * @custom:security-contact security@inverter.network
+ *                          In case of any concerns or findings, please refer
+ *                          to our Security Policy at security.inverter.network
+ *                          or email us directly!
+ *
+ * @custom:version 1.0.0
+ *
+ * @author  Inverter Network
+ */
+contract FM_Template_v1 is IFM_Template_v1, Module_v1 {
+    //==========================================================================
+    // Libraries
+
+    using SafeERC20 for IERC20;
+
+    //==========================================================================
+    // ERC165
+
+    /// @inheritdoc ERC165Upgradeable
+    function supportsInterface(bytes4 interfaceId_)
+        public
+        view
+        virtual
+        override(Module_v1)
+        returns (bool)
+    {
+        return interfaceId_ == type(IFM_Template_v1).interfaceId
+            || interfaceId_ == type(IFundingManager_v1).interfaceId
+            || super.supportsInterface(interfaceId_);
+    }
+
+    //==========================================================================
+    // Constants
+
+    // Add constants here
+
+    //==========================================================================
+    // State
+
+    /// @dev    Mapping of user addresses to their deposited token amounts.
+    mapping(address user => uint amount) internal _depositedAmounts;
+
+    /// @dev    The orchestrator token.
+    IERC20 internal _orchestratorToken;
+
+    /// @dev    Storage gap for future upgrades.
+    uint[50] private __gap;
+
+    //==========================================================================
+    // Modifiers
+
+    // Add modifiers here
+
+    //==========================================================================
+    // Constructor & Init
+
+    /// @inheritdoc Module_v1
+    function init(
+        IOrchestrator_v1 orchestrator_,
+        Metadata memory metadata_,
+        bytes memory configData_
+    ) external override(Module_v1) initializer {
+        __Module_init(orchestrator_, metadata_);
+
+        // Decode module specific init data through use of configData bytes.
+        // This value is an example value used to showcase the setters/getters
+        // and internal functions/state formatting style.
+        (address orchestratorTokenAddress) = abi.decode(configData_, (address));
+
+        // Set init state.
+        _orchestratorToken = IERC20(orchestratorTokenAddress);
+    }
+
+    //==========================================================================
+    // Public (Getters)
+
+    /// @inheritdoc IFM_Template_v1
+    function getDepositedAmount(address user_)
+        external
+        view
+        virtual
+        override
```

Comment:
Are you overriding anything here? Same for all other functions: override shouldn't be used unless we are overriding anything. The important part is the virtual keyword.